src/os/windows/win32.h

```diff
@@ -10,50 +10,9 @@
 #ifndef WIN32_H
 #define WIN32_H
 
-#include <windows.h>
-bool MyShowCursor(bool show, bool toggle = false);
-
-class DllLoader {
-public:
-	explicit DllLoader(LPCTSTR filename)
-	{
-		this->hmodule = ::LoadLibrary(filename);
-		if (this->hmodule == nullptr) this->success = false;
-	}
-
-
-	~DllLoader()
-	{
-		::FreeLibrary(this->hmodule);
-	}
-
-	bool Success() { return this->success; }
+#include <string>
```

Comment:
This shouldn't be needed.

Reply:
Why not? It very much is :)

This header file uses `std::string_view`, defined in `<string>`. Realise, as I mentioned in the description, that there used to be a `#include <windows>` in this file. It was hiding all these dependencies.

Reply:
Ah, this is in reference to an earlier comment I guess. Sure.

Reply:
Yeah commonly used std headers are included in stdafx.h for precompiled headers